home.js

```diff
@@ -0,0 +1,116 @@
+const ham = document.querySelector('.hamBtn');
+const closeBtn = document.querySelector('.closeBtn');
+const mobMenu = document.querySelector('.mob-menu');
+
+ham.addEventListener('click', () => {
+  mobMenu.style.display = 'flex';
+  // /*body.style.filter = 'blur(100px)';*/
+});
+
+closeBtn.addEventListener('click', () => {
+  mobMenu.style.display = 'none';
+  // /*body.style.filter = 'blur(0)';*/
+});
+
+mobMenu.addEventListener('click', () => {
+  mobMenu.style.display = 'none';
+  // /*body.style.filter = 'blur(0)';*/
+});
+
+const speakers = [
+  {
+    id: '1',
+    name: 'Ms Phoina Beauty',
+    title: 'CEO Phoina Hair Collection',
+    desc: 'Aisha is a beauty enthusiast and make-up artist with passion to educate young women into beauty.',
+    img: './Logo/phoina.jpg',
+    imgAlt: 'Phoina Beauty',
+  },
+  {
+    id: '2',
+    name: 'Mrs Maureen Bandari',
+    title: 'CEO Bandari Beauty',
+    desc: 'Maureen Bandari is an entrepreneur with focus on skin beauty.She is the founder of Badari Beauty Kenya.',
+    img: './Logo/bandari.jpg',
+    imgAlt: 'Maureen Bandari',
+  },
+  {
+    id: '3',
+    name: 'Mrs Wanjiru Kariuki',
+    title: 'CEO Maua wear',
+    desc: 'Wanjiru Kariuki is a digital content creator and a fashion enthusiast. She is currently the founder of Maua wear-a made Kenya brand',
+    img: './Logo/wanj.jpg',
+    imgAlt: 'Wanjiru Kariuki',
+  },
+  {
+    id: '4',
+    name: 'Ms Jackie Matubia',
+    title: 'CEO Afrique Salon',
+    desc: 'Jackie Matubia is an influencer with good taste of fashion and beauty.She is the founder of Afrique salon',
+    img: './Logo/jackie.jpg',
+    imgAlt: 'Jackie Matubia',
+  },
+  {
+    id: '5',
+    name: 'Mrs Dee Neta',
+    title: 'CEO Fepa Beauty',
+    desc: 'Dee Neta is a digital creator and the founder of Fepa Beauty.She also has her own beauty line with her own formulated beauty care products.',
+    img: './Logo/dee.jpg',
+    imgAlt: 'Dee Neta',
+  },
+  {
+    id: '6',
+    name: 'Ms Zedik Posh',
+    title: 'CEO Posh palace beauty',
+    desc: 'Zedik Posh is a content creator and founder of Posh palace.She offers trainings to anyone with a passion of beauty.',
+    img: './Logo/zedik.jpg',
+    imgAlt: 'Zedik Posh',
+  },
+];
+
+const speakerContainer = document.querySelector('.speaker-holder');
+const showSpeakers = speakers.map((speaker) => `
+      <div class="speakers">
+        <div class="img-holder">
+          <img class="bg" src="">
+          <img src=${speaker.img} alt=${speaker.imgAlt}>
+        </div>
+        <div class="text-holder">
+          <div class="name">
+            <h3>${speaker.name}</h3>
+            <i>${speaker.title}</i>
+          </div>
+          <hr>
+          <div class="description">
+            ${speaker.desc}  
+          </div>
+        </div>
+      </div>
+      `).join('');
```

Comment:
- [ ] This is a beautiful and very maintainable implementation :100: but we just need to pay closer attention to the styling and arrangement. Please mind the design blueprint from Cindy Shin for views - how the images have equal sizes, the same start points, and texts having a left alignment or a flex-start alignment if using a flex display.
- In case you need to see how it looks:
#Desktop

|Your Design| Cindy Shin's|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/251868410-22da26a6-49c5-4d7c-9f56-a017f2e027fb.png)|![image](https://user-images.githubusercontent.com/99715640/251868436-475a4db5-7d05-4cd1-8c2e-b5ed56d30422.png)|

##Mobile

|Your Design| Cindy Shin's|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/251869275-3f3c9054-706e-4dbd-98b0-f60ae0c991de.png)|![image](https://user-images.githubusercontent.com/99715640/251869366-b7e9f867-a711-4344-9bf1-a262e0d9550c.png)|